pkg/download/mode/mode.go

```diff
@@ -53,8 +85,8 @@ type DownloadPath struct {
 func NewFile(m Mode, downloadURL string) (*DownloadFile, error) {
 	const op errors.Op = "downloadMode.NewFile"
 
-	if m == "" {
-		return nil, errors.E(op, downloadModeErr)
+	if err := m.Validate(); err != nil {
```

Comment:
I think this error is now less readable, because before it used to say `download mode is not set` but now it will say ` isn't a valid value` (notice the space in the beginning because there's nothing) 

I think in the Sprintf of the validate function we should pass a `%q` so it would say `"" isn't a valid value` or we should keep the old error here for better messaging.

Reply:
How about if we pass `%q` as you said, and also prefix the error with "Download mode ..."? 
The result would then be `Download mode "" isn't a valid value`

Reply:
That sounds great